### ISSUE TITLE
i'can't list VRF created apart from default vrf

### ISSUE THREAD

**Issue by any4one** [SpriteLink/NIPAP#1308]
Hi team, please. I created graphically 4 new vrfs in addition to the new one. Now when i want to create a new prefix and assign a vrf, the only vrf displayed is the default one. I need your help please.
![2022-06-23 10_35_28-IPAM - VMware Workstation](https://user-images.githubusercontent.com/21181551/175255851-95f1ba7f-d0fe-4117-884f-9b6656e07a33.png)

**Comment by garberg** [comment]
Right, you need to search for the VRFs using the highlighted text box.
